scripts/build/llvm-suffix.sh

```diff
@@ -0,0 +1,58 @@
+# This file is meant to be included by shell scripts
```

Comment:
You could mention somewhere, that these LLVM suffixes represent different build options (which ones).

Reply:
Good point.